packages/dds/tree/src/feature-libraries/sequence-field/format.ts

```diff
@@ -220,6 +220,7 @@ export interface Revive<TNodeChange = NodeChangeType>
 		HasChanges<TNodeChange>,
 		CanConflict {
 	type: "Revive";
+	content: ProtoNode[];
```

Comment:
I was hoping we could store an `ITreeCursorSynchronous[]` instead. Otherwise it means we're converting back and forth between the durable cursor and the JsonableTree format.
See if you can make it work with `ITreeCursorSynchronous[]`.